web-client/public/js/update-app.js

```diff
@@ -491,7 +500,7 @@ const toggleLayout = (on, off) => {
 };
 
 const updatetoForceGraph = () => {
-    $(LOCK_SLIDERS_BUTTON).removeAttr("disabled");
+    // $(LOCK_SLIDERS_BUTTON).removeAttr("disabled");
     console.log("Update to Force graph");
```

Comment:
If we really don’t need the commented-out line anymore, it’s OK to delete it outright (GitHub has it anyway in the history)

Typically we also reserve console logs for debugging. I notice that this will leave the function empty; I think that’s OK, just in case we need to add code in the future when `updateToForceGraph` takes place